microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -62,12 +63,16 @@ def verify_mshvlog_is_active(
         self._save_dmesg_logs(node, log_path)
         mshvlog_running = node.tools[Service].is_service_running("mshvlog")
         if not mshvlog_running:
-            log.error("mshvlog service is not running on MSHV root partition.")
+            log.error(
+                "mshvlog service is not running on MSHV root \
+                      partition."
+            )
```

Comment:
why change this?

Reply:
This is mostly a formatting change. Flake8 reported a failure as the line was longer.

Reply:
@LiliDeng Could you please approve this PR, if you do not have any other concerns?

Reply:
@praveen-pk no need change it, the max length is 89 characters.

Reply:
I reverted this formatting change.